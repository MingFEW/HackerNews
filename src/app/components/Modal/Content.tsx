import React from 'react'
import 'twin.macro'

interface Props {
  children: React.ReactNode
}

const Content: React.FC<Props> = ({ children }) => {
  return <div tw="p-4 overflow-auto bg-card [height: calc(100vh - 59px - 48px)]">{children}</div>
}

export default Content
