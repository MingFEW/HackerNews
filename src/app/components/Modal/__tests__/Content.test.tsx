import { createRenderer } from 'react-test-renderer/shallow'

import Content from '../Content'

const shallowRenderer = createRenderer()

describe('<Content />', () => {
  it('should render and match the snapshot', () => {
    shallowRenderer.render(<Content>content</Content>)
    const renderedOutput = shallowRenderer.getRenderOutput()
    expect(renderedOutput).toMatchSnapshot()
  })
})
