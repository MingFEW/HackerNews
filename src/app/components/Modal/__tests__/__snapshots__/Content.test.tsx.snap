// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Content /> should render and match the snapshot 1`] = `
<div
  tw="p-4 overflow-auto bg-card [height: calc(100vh - 59px - 48px)]"
>
  content
</div>
`;
